### DISCUSSION TITLE
Mod to start song at different time

### DISCUSSION THREAD

**Discussion by rasputozen** [ppy/osu#15099] [Ideas] [answered]
Sometimes I encounter a difficult passage in the middle or end of a song I really like and it's a pain to go into the editor, create a new map that starts at that point, practice the passage on this new map, and then delete it when I'm done to avoid clutter. Would be nice to have a mod with a slider that lets me set an offset from the start.

**Comment by Semro** [comment, marked as answer]
Seems like a duplicate of https://github.com/ppy/osu/discussions/13013
